backend/driver/driver.go

```diff
@@ -108,9 +108,18 @@ func (d *driverImpl) Run() error {
 	if err := d.updateStatus(lmevalservicev1beta1.RunningJobState); err != nil {
 		return err
 	}
-	err := d.exec()
+	execErr := d.exec()
 
-	return d.updateCompleteStatus(err)
+	// dump stderr and stdout to the console
```

Comment:
Is this just an additional way of posting the output of the job itself?

Reply:
I found that it's easier to debug if there is anything wrong while running the job. Before the change, there are only log messages from the driver. The job's results are populated into LMEvalJob's status. If you want me to dump the results to the console, I can do that too.